Python/bytecodes.c

```diff
@@ -3754,6 +3772,7 @@ dummy_func(
             _PyThreadState_PopFrame(tstate, frame);
             frame = tstate->current_frame = prev;
             _PyFrame_StackPush(frame, (PyObject *)gen);
+            LOAD_IP(frame->return_offset);
```

Comment:
Isn't `frame->return_offset` always 0 here?

Reply:
why? this should be like any return

Reply:
We've just created the generator in `_Py_MakeCoro` above, and nothing could have changed `return_offset`.

Reply:
This is not the same frame.

Reply:
Ah, yes. My mistake.